client-server/phoebe-server

```diff
@@ -584,8 +578,8 @@ def new_bundle(type):
     type: 'binary:detached'
     """
     j = _get_request(request)
-    clientid = j.get('clientid', None) if j is not None else None
-    client_version = j.get('client_version', None) if j is not None else None
+    #clientid = j.get('clientid', None) if j is not None else None
```

Comment:
if adhering to style, then we should add a space after the comment?

Reply:
I'm not trying to _completely_ adhere to style (yet), but can make this update